src/Compilers/Core/Portable/Operations/OperationCloner.cs

```diff
@@ -47,7 +47,13 @@ internal override IOperation VisitNoneOperation(IOperation operation, object arg
         private ImmutableArray<T> VisitArray<T>(ImmutableArray<T> nodes) where T : IOperation
         {
             // clone the array
-            return nodes.SelectAsArray(n => Visit(n));
+            return nodes.IsDefault ? default : nodes.SelectAsArray(n => Visit(n));
```

Comment:
>IsDefault [](start = 25, length = 9)

Are default arrays allowed in operations? I suspect that empty arrays are used presently.